rocketpy/Rocket.py

```diff
@@ -681,26 +717,36 @@ def cnalfa1(cn):
             )
 
             # Takes an approximation to an angular coefficient
-            clalpha = cldata.differentiate(x=0, dx=1e-2)
-
-            # Parameters for Roll Moment. Documented at: https://github.com/Projeto-Jupiter/RocketPy/blob/develop/docs/technical/aerodynamics/Roll_Equations.pdf
-            clfDelta = n * (Ymac + radius) * clalpha / d
-            cldOmega = (
-                n * clalpha * np.cos(cantAngleRad) * trapezoidalConstant / (Af * d)
-            )
-            rollParameters = (
-                [clfDelta, cldOmega, cantAngleRad] if cantAngleRad != 0 else [0, 0, 0]
-            )
+            clalpha = cl.differentiate(x=0, dx=1e-2)
+
+        # Parameters for Roll Moment.
+        # Documented at: https://github.com/Projeto-Jupiter/RocketPy/blob/develop/docs/technical/aerodynamics/Roll_Equations.pdf
+        clfDelta = (
+            rollForcingInterferenceFactor * n * (Ymac + radius) * clalphaSingleFin / d
+        )  # Function of mach number
+        cldOmega = (
+            2
+            * rollDampingInterferenceFactor
+            * n
+            * clalphaSingleFin
+            * np.cos(cantAngleRad)
+            * trapezoidalConstant
+            / (Aref * d ** 2)
+        )
+        # Function of mach number
+        rollParameters = (
+            [clfDelta, cldOmega, cantAngleRad] if cantAngleRad != 0 else [0, 0, 0]
+        )
```

Comment:
```suggestion
        rollParameters = (
            [clfDelta, cldOmega, cantAngleRad] if cantAngleRad != 0 else [Function(0), Function(0), 0]
        )
```

The current implementation causes a fatal error when running any simulation without a cant angle, since we get an error in Flight:1351:
```python
if aerodynamicSurface[-1] == "Fins":
        Clfdelta, Cldomega, cantAngleRad = aerodynamicSurface[2]
        M3f = (
            (1 / 2 * rho * freestreamSpeed ** 2)
            * Aref
            * d
            * Clfdelta(freestreamMach)
            * cantAngleRad
        )
```

If `Clfdelta` is 0, then calling `Clfdelta(freestreamMach)` throws an error.